### ISSUE TITLE
Can you guys please please add support for Deno deployment

### ISSUE THREAD

**Issue by Evertt** [telegraf/telegraf#1812]
It's possible to import telegraf in Deno using `import { Telegraf } from "npm:telegraf@4.12.2"`.
I mean when I run that locally, that works.
But for some reason that fails when trying to deploy it on Deno.

And then there are Deno friendly CDNs, and I've tried all of them with Telegraf, and all of them encounter an error for one reason or another.

Can you maybe please just make a port that is completely compatible with Deno, including its deployment platform?

**Comment by MKRhere** [comment]
What happens when you Deno Deploy with `npm:telegraf`?

In other news: Telegraf v5 will natively support Deno (WIP in [feat-deno](https://github.com/telegraf/telegraf/tree/feat-deno))

**Comment by Evertt** [comment]
> What happens when you Deno Deploy with `npm:telegraf`?


It just so happens that importing npm packages that way is not yet supported on Deno deploy.


> In other news: Telegraf v5 will natively support Deno (WIP in [feat-deno](https://github.com/telegraf/telegraf/tree/feat-deno))

Oh really?! That's very exciting! What's the expected time-line?

**Comment by MKRhere** [comment]
> npm packages that way is not yet supported

Right; would you be able to report the errors you faced with https://esm.sh/telegraf?

> What's the expected time-line?

Unfortunately no real timeline yet; it's very close to ready, but announcing a date puts a lot of pressure on me, and I have other things to work on as well. But relatively soon!

**Comment by Evertt** [comment]
I'm sorry I didn't respond anymore. I switched over to using a docker image from Deno and deploying that on Google Cloud and now I don't have to worry anymore about any restrictions that the Deno deployment environment has.

And uhm, I have a quite a lot on my plate right now, so I don't really feel for going back to check out what errors came out of esm.sh when trying to install telegraf. Sorry. 😓

**Comment by MKRhere** [comment]
No problem! We'll try to get the first alpha for v5 out in the coming weeks!
